### PR TITLE
Added node ASG update policy parameters

### PR DESCRIPTION
**Pull request by micahhausler**
*Issue #, if available:*

N/A

*Description of changes:*

Added node ASG update parameters

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: amazon-eks-nodegroup.yaml [hunk 1]
```diff
@@ -155,6 +175,10 @@ Metadata:
           - NodeGroupName
           - NodeAutoScalingGroupMinSize
           - NodeAutoScalingGroupMaxSize
+          - NodeAutoScalingGroupDesiredCapacity
+          - NodeAutoScalingGroupUpdatePolicyMaxBatchSize
+          - NodeAutoScalingGroupUpdatePolicyMinInstancesInService
+          - NodeAutoScalingGroupUpdatePolicyPauseTime
```
Comment (micahhausler) [inline amazon-eks-nodegroup.yaml]: This is getting a little dense, any opinions about moving the UpdatePolicy parameters to their own ParameterGroup for UI cleanliness?

File: amazon-eks-nodegroup.yaml [hunk 2]
```diff
@@ -107,8 +107,28 @@ Parameters:
...
+
+  NodeAutoScalingGroupUpdatePolicyMinInstancesInService:
+    Type: Number
+    Description: MinInstancesInService for the NodeGroup UpdatePolicy
+    Default: 1
+
+  NodeAutoScalingGroupUpdatePolicyPauseTime:
+    Type: String
+    Description: PauseTime for the NodeGroup UpdatePolicy
+    Default: "PT0S"
```
Comment (nckturner) [inline amazon-eks-nodegroup.yaml]: What is PT0S?  Do we need some documentation on what values are accepted here?

Comment (micahhausler) [reply amazon-eks-nodegroup.yaml]: It is the default, zero value. Added a doc url.

File: amazon-eks-nodegroup.yaml (hunk 1 above)
Comment (mogren) [reply amazon-eks-nodegroup.yaml]: Maybe all 6 ASG settings in that case.

**Pull request merged**